searches/ternary_search.py

```diff
@@ -69,35 +148,28 @@ def rec_ternary_search(left, right, A, target):
             return rec_ternary_search(left, oneThird - 1, A, target)
         elif A[twoThird] < target:
             return rec_ternary_search(twoThird + 1, right, A, target)
-
         else:
             return rec_ternary_search(oneThird + 1, twoThird - 1, A, target)
     else:
-        return None
+        return -1
 
 
 # This function is to check if the array is sorted.
 def __assert_sorted(collection):
     if collection != sorted(collection):
-        raise ValueError("Collection must be sorted")
+        raise ValueError("Collection is not sorted.")
     return True
 
 
 if __name__ == "__main__":
-    user_input = input("Enter numbers separated by coma:\n").strip()
-    collection = [int(item) for item in user_input.split(",")]
-
-    try:
-        __assert_sorted(collection)
-    except ValueError:
-        sys.exit("Sequence must be sorted to apply the ternary search")
-
-    target_input = input("Enter a single number to be found in the list:\n")
-    target = int(target_input)
+    user_input = input("Enter numbers separated by comma:\n").strip()
+    collection = [int(item.strip()) for item in user_input.split(",")]
```

Comment:
```suggestion
    collection = sorted(int(item.strip()) for item in user_input.split(","))
```
We can sort the collection and remove the assertion logic.